README.md

```diff
@@ -1,3 +1,9 @@
+# Deprecation
+
+:rotating_light: **Assets Frontend has been deprecated in favour of the [Play Frontend](https://confluence.tools.tax.service.gov.uk/display/PLATUI/play-frontend+Libraries).** :rotating_light:
```

Comment:
As this is a public repo, we might want to link to https://www.github.com/hmrc/play-frontend-hmrc README instead?

Reply:
I considered that but as that's only one of the libraries we're pointing users to I felt it was better to use the documentation which is most like a landing-page.

Reply:
Cool, it's worth bearing in mind this link won't be accessible outside HMRC but I guess this isn't going to be an issue for most. Historically I believe internal URLs weren't allowed in public repos but I think that rule might have been relaxed recently.